ui/src/app/pages/profile/profile-registration-step-status.tsx

```diff
@@ -60,48 +62,42 @@ const ProfileRegistrationStepStatus: React.FunctionComponent<Props> =
       incompleteButtonText,
       completedButtonText,
       completeStep,
-      completionTimestamp,
       children
     } = props;
 
     if (environment.enableProfileCapsFeatures) {
       return (
-        <div style={styles.container}>
+        <FlexColumn style={{...styles.container, ...props.containerStylesOverride}}>
           <div style={styles.title}>
             { title }
           </div>
-          <FlexColumn>
-            <div>
-              { isComplete ? (
-                <FlexRow>
-                  <FlexColumn>
-                    <div>{ wasBypassed ? 'Bypassed By Admin on:' : completedButtonText + ' on:' }
-                    </div>
-                    <div>{new Date(completionTimestamp).toDateString()}</div>
-                    <Button disabled={true} data-test-id='completed-button'
-                            style={{backgroundColor: colors.success,
-                              width: 'max-content', marginTop: '1rem',
-                              cursor: 'default'}}>
-                      <ClrIcon shape='check' style={{marginRight: '0.3rem'}}/>{completedButtonText}
-                    </Button>
-                  </FlexColumn>
+          <FlexColumn style={{justifyContent: 'space-between', flex: '1 1 auto'}}>
 
-                </FlexRow>
-              ) : (
+            { isComplete ? (
+              <React.Fragment>
+                <div style={props.childrenStyle}>
+                  { children }
+                </div>
+                <Button disabled={true} data-test-id='completed-button'
+                        style={{...styles.button, backgroundColor: colors.success,
+                          width: 'max-content', cursor: 'default'}}>
+                  <ClrIcon shape='check' style={{marginRight: '0.3rem'}}/>{wasBypassed ? 'Bypassed' : completedButtonText}
+                </Button>
+              </React.Fragment>
+            ) : (
+              <React.Fragment>
```

Comment:
curious can we do like this <React.Fragment>isComplete && </React.Fragment>?

Reply:
Almost certainly. Let me see!

Reply:
Actually, I don't think we can on thinking more. Because react expects a single element to be returned from a function (which is why React.Fragment exists), so I think it would be unhappy with multiple elements being returned.